lib/deck.rb

```diff
@@ -0,0 +1,19 @@
+class Deck
+  attr_reader :cards, :count
+  def initialize (cards)
+    @cards = cards
+    @count = cards.count
+  end
+
+  def cards_in_category(category)
+    same_category = []
+    @cards.each do |card|
+      if card.category == category
+        same_category << card
+      end
+    end
+    same_category
```

Comment:
great job here! We haven't talked much about other enumerables yet, but this could be a great method for using `find_all`.